src/ngSanitize/sanitize.js

```diff
@@ -476,6 +477,7 @@ function $SanitizeProvider() {
           out(tag);
           out('>');
         }
+        // eslint-disable-next-line eqeqeq
```

Comment:
>
> In src/ngSanitize/sanitize.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72962101>:
>
> > @@ -476,6 +477,7 @@ function $SanitizeProvider() {
> >            out(tag);
> >            out('>');
> >          }
> > +        // eslint-disable-next-line eqeqeq
>
> I am pretty sure this one could be converted to ===.
>
This is a sensitive file so I was afraid of touching it in this way. A next
security review would be necessary; if a few people say it's safe, I may
change it.